pkgs/build-support/dotnet/build-dotnet-module/default.nix

```diff
@@ -138,25 +138,49 @@ in stdenvNoCC.mkDerivation (args // {
       exclusions = dotnet-sdk.passthru.packages { fetchNuGet = attrs: attrs.pname; };
     in writeShellScript "fetch-${pname}-deps" ''
       set -euo pipefail
-      export PATH="${lib.makeBinPath [ coreutils dotnet-sdk nuget-to-nix ]}"
+      export PATH="${lib.makeBinPath [ coreutils dotnet-sdk nuget-to-nix ]}":$PATH
 
+      deps_file="$(realpath "''${1:-/tmp/${pname}-deps.nix}")"
       cd "$(dirname "''${BASH_SOURCE[0]}")"
 
+      while (( $# > 0 )); do
+        case "$1" in
+          --post-unpack)
```

Comment:
Seeing what you use this flag for, I dont think its needed. I encountered a similar problem in `eventstore` - thats why I changed the update script to use `srcOnly`.

Then, if you put the post unpack changes in the `postPatch` hook, they will modify the source used by both the update script and the actual build. Then we have the added benefit of deduplication.

Unless I missed something, all usages of `--post-unpack` would be better with putting them in `postPatch`

Reply:
I had completely forgotten about that :sweat_smile: 

I reverted those changes, it indeed was not required at all. The only derivation that used it (`omnisharp-roslyn`) already worked out of the box.